package.json

```diff
@@ -9,6 +9,12 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.0",
+    "@emotion/styled": "^11.10.0",
+    "@fontsource/roboto": "^4.5.8",
```

Comment:
No need for separate package, just use this:
https://mui.com/material-ui/getting-started/installation/#roboto-font

Reply:
done